.github/workflows/helm_tests.yml

```diff
@@ -40,7 +40,7 @@ jobs:
         with:
           workloads: "" # all
           namespace: "" # default
-          timeout: 300
+          timeout: 1000
```

Comment:
for whatever reason, installing `grpcio` takes forever so I needed to increase this timeout.. (`grpcio` is used by the `firebase_admin` package)